mlflow/tracing/utils/copy.py

```diff
@@ -0,0 +1,46 @@
+from typing import Any, Optional
+
+from mlflow.entities.span import LiveSpan, Span
+from mlflow.tracing.trace_manager import InMemoryTraceManager
+
+
+def copy_trace_to_experiment(
+    trace_dict: dict[str, Any], experiment_id: Optional[str] = None
+) -> str:
+    """
+    Copy the given trace to the current experiment.
+    The copied trace will have a new trace ID and location metadata.
+
+    Args:
+        trace_dict: The trace dictionary returned from model serving endpoint.
+            This can be either V2 or V3 trace.
+        experiment_id: The ID of the experiment to copy the trace to.
+            If not provided, the trace will be copied to the current experiment.
+    """
+    new_trace_id, new_root_span = None, None
```

Comment:
```suggestion
    new_trace_id = None
    new_root_span = None
```

let's split this t avoid unnecessary tuple creation and unpacking.